### PR TITLE
Bump tiller version to 2.13.1

### PR DESCRIPTION
**Pull request by chenrui333**
Just update the tiller image version as well as the release tag in the README.

### CODE REVIEW

**Comment by rporres** [comment]
@pdecat Since you updated helm dependencies against `master` #261, can you take a look into this? I guess it makes sense to have the latest tiller although helm `master` version is still `v2.12`

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,11 @@
 ## 0.9.2 (Unreleased)
```
Comment (rporres) [inline CHANGELOG.md]: Please remove this part. We'll do it when we cut a new release.

File: README.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ distributions from the [Releases](https://github.com/terraform-providers/terrafo
 Download and uncompress the latest release for your OS. This example uses the linux binary.
 
 ```sh
-> wget https://github.com/terraform-providers/terraform-provider-helm/releases/download/v0.7.0/terraform-provider-helm_v0.7.0_linux_amd64.tar.gz
```
Comment (rporres) [inline README.md]: This is out of the scope of this change

File: .travis.yml [hunk 3]
```diff
@@ -40,15 +40,15 @@ jobs:
         - sudo mount --make-rshared /
         - curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/${KUBERNETES_VERSION}/bin/linux/amd64/kubectl
         - sudo chmod +x kubectl && sudo mv kubectl /usr/local/bin/
-        - curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
```
Comment (rporres) [inline .travis.yml]: This change does not belong to this PR description

**Review by rporres — CHANGES_REQUESTED** [review]
Thx for your PR

A few minor comments, just to make changes be aligned with the PR description

File: CHANGELOG.md (hunk 1 above)
Comment (chenrui333) [reply CHANGELOG.md]: Sure, sounds good!

File: README.md (hunk 2 above)
Comment (chenrui333) [reply README.md]: sure, I will submit a separate PR to fix this release reference issue as well. 

Let me know if that works.

File: .travis.yml (hunk 3 above)
Comment (chenrui333) [reply .travis.yml]: that just came along with formatter, I can fix this as well.

**Comment by chenrui333** [comment]
@rporres updated the PR, thanks!

**Review by pdecat — APPROVED** [review]
LGTM!
